src/CommunityToolkit.Maui/Views/AvatarView.shared.cs

```diff
@@ -72,6 +72,8 @@ public AvatarView()
 			CornerRadius = new CornerRadius(AvatarViewDefaults.DefaultCornerRadius.TopLeft, AvatarViewDefaults.DefaultCornerRadius.TopRight, AvatarViewDefaults.DefaultCornerRadius.BottomLeft, AvatarViewDefaults.DefaultCornerRadius.BottomRight),
 		};
 		Content = avatarLabel;
+		avatarImage.SetBinding(WidthRequestProperty, new Binding(nameof(WidthRequest), source: this));
+		avatarImage.SetBinding(HeightRequestProperty, new Binding(nameof(HeightRequest), source: this));
```

Comment:
These need to be bound to `Parent.WidthRequest` and `Parent.HeightRequest`. Currently, they are just binding to themselves via `source: this`.

```suggestion
		avatarImage.SetBinding(WidthRequestProperty, new Binding(nameof(WidthRequest), source: Parent));
		avatarImage.SetBinding(HeightRequestProperty, new Binding(nameof(HeightRequest), source: Parent));
```